### PR TITLE
Import with MySQL CLI

### PR DESCRIPTION
**Pull request by hooopo**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/_index.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ hide_commit: true
 
 [From Apache Parquet Files](https://docs.pingcap.com/tidbcloud/import-csv-files)
 
+[From MySQL CLI](https://docs.pingcap.com/tidbcloud/import-with-mysql-cli)
```
Comment (ran-huang) [inline tidb-cloud/_index.md:76]:
```suggestion
[With MySQL CLI](https://docs.pingcap.com/tidbcloud/import-with-mysql-cli)
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+# Import Data into TiDB Cloud via MySQL CLI
+
+This document describes how to import data into TiDB Cloud via the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html). You can import data from an SQL file or a CSV file. The following sections provide step-by-step instructions for importing data from each type of file.
+
+## Prerequisites
+
+Before you can import data via MySQL CLI to TiDB Cloud, you need the following prerequisites:
+
+- You have access to your TiDB Cloud Cluster. If you do not have a TiDB cluster, create one following the instructions in [Build a TiDB Serverless Cluster](/develop/dev-guide-build-cluster-in-cloud.md).
```
Comment (ran-huang) [inline tidb-cloud/import-with-mysql-cli.md:14]:
```suggestion
- You have access to your TiDB Cloud cluster. If you do not have a TiDB cluster, create one following the instructions in [Build a TiDB Serverless Cluster](/develop/dev-guide-build-cluster-in-cloud.md).
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+Do the following to import data from an SQL file:
+
+1. Provide a real SQL file (for example, `product_data.sql`) that contains the data you want to import. This SQL file must contain `INSERT` statements with real data.
+
+2. Use the following command to import data from the SQL file:
+
+```bash
+mysql --comments --connect-timeout 150 -u '<your_username>' -h <your_cluster_host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=<your_ca_path> -p <your_password> < product_data.sql
+```
```
Comment (ran-huang) [inline tidb-cloud/import-with-mysql-cli.md:88]:
```suggestion
    ```bash
    mysql --comments --connect-timeout 150 -u '<your_username>' -h <your_cluster_host> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=<your_ca_path> -p <your_password> < product_data.sql
    ```
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+---
+title: Import Data into TiDB Cloud via MySQL CLI
+summary: Learn how to import Data into TiDB Cloud via Import Data into TiDB Cloud via MySQL CLI.
```
Comment (ran-huang) [inline tidb-cloud/import-with-mysql-cli.md:3]:
```suggestion
summary: Learn how to import Data into TiDB Cloud via MySQL CLI.
```

File: TOC-tidb-cloud.md
Comment (ran-huang) [inline TOC-tidb-cloud.md:1]: preview: https://pingcap-docs-website-scaffold-git-preview-clou-cff3c4-ran-huang.vercel.app/tidbcloud/import-with-mysql-cli

**Review by ran-huang — APPROVED** [review]

**Pull request merged**